Tests/Hotels/Homepage.feature

```diff
@@ -0,0 +1,57 @@
+@signUpHotels
+Feature: Homepage
+
+  @TC-28
+  Scenario: Verify Child-age dropdowns are same as number of Children selected
+    When I Lunch hotels.com homepage
+    And I click on Travelers
+    And I Select “Children” as 2
+    Then I Verify Children-age dropdown are 2
+    And I Verify Plus Button is enabled
+    And I Verify Minus Button is enabled
+    When I Select “Children” as 6
+    Then I Verify Children-age dropdown are 6
+    And I Verify Plus Button is disabled
+    And I Verify Minus Button is enabled
+    When I Select “Children” as 5
+    Then I Verify Children-age dropdown are 5
+    And I Verify Plus Button is enabled
+    And I Verify Minus Button is enabled
+    When I Select “Children” as 0
+    Then I Verify Children-age dropdown is NOT displayed
+    And I Verify Plus Button is enabled
+    And I Verify Minus Button is disabled
+
+  @TC-18
+  Scenario: Verify user can update number of guests on Home page
+    When I Lunch hotels.com homepage
+    Then I click on Travelers
+    Then I Select “Adults as 6
+    And I Select “Children” as 3
+    And I Select first child age: 4
+    And I Select second child age: Under 1
+    And I Select third child age: 7
+    Then I Click Done
+    Then I Verify total number of Travelers is sum of adults and children as same as selected on step #3 and #4
```

Comment:
You forgot to update Gherkin labels for other scenarios. Pls follow the way we did for first scenario.